example/django/README.md

```diff
@@ -1,69 +1,44 @@
 # Bugsnag Django demo
 
-This Django application demonstrates how to use Bugsnag with the Django web
-framework for Python. Before testing it, open up the `bugsnag_demo/settings.py`
-file and configure your API key (see the very bottom of the file).
+This Django application demonstrates how to use Bugsnag with the Django web framework for Python.
 
-```
-BUGSNAG = {
-    "api_key": "066f1ad3590596f9aacd601ea89af845"
-}
-```
+Please note this particular example app will only work with Python 3+ and Django 2+, to showcase the most current configurations. Check out [this example](https://github.com/bugsnag/bugsnag-python/example/django1_10) to see the configuration for  Python 2 and Django 1.
 
-Install dependencies.
+## Setup
 
-```
-pip install -r requirements.txt
-```
+Try this out with [your own Bugsnag account](https://app.bugsnag.com/user/new)! You'll be able to see how the errors are reported in the dashboard, how breadcrumbs are left, how errors are grouped and how they relate to the original source.
 
-This minimal setup is enough to start Bugsnagging.
+To get set up, follow the instructions below. Don't forget to replace the placeholder API token with your own!
 
-Run the application.
 
-```
-python manage.py runserver
-```
+1. Clone the repo and `cd` into this directory:
+    ```shell
+    git clone https://github.com:bugsnag/bugsnag-python.git
+    cd bugsnag-python/example/django
+    ```
 
-Next, open your project's dashboard on Bugsnag.
+1. Install dependencies
+    ```shell
+    pip install -r requirements.txt
+    ```
 
-1. [crash](/crash)
-<br/>
-Crashes the library and sends a notification about the nature of the crash.
-Basically, almost any unhandled exception sends a notification to Bugsnag.
-Pressing this link would lead to an empty page, which is normal.
-See `demo/views.py, #crash`
+1. Before testing it, open up the [`settings.py`](bugsnag_demo/settings.py)
+    file and configure your API key.
```

Comment:
I opened up this file and saw `SECRET_KEY`, which could potentially be a bit confusing. Could we define a similar constant for the API key at the top of the file also?

Reply:
That's a Django feature. To avoid confusion I could modify to:
```
Before testing it, open up the [`settings.py`](bugsnag_demo/settings.py)
file and configure your API key within the `BUGSNAG` dictionary.
```